.babelrc

```diff
@@ -1,12 +1,20 @@
 {
     "presets": [
-    "es2015",
-    "stage-0"
-  ],
-  "plugins": [
-    ["transform-runtime", {
-      "polyfill": false,
-      "regenerator": true
-    }]
-  ]
+        [
+            "@babel/preset-env",
+            {
+                "targets": {
+                    "browsers": [
+                        "last 2 versions"
+                    ]
+                },
+                "debug": true,
```

Comment:
do you really need debug?

additionally babel can be configured for specific env like for dev debug is on, for prod it's off